### PR TITLE
Change url in the docusaurus demo config

### PR DESCRIPTION
**Pull request by JoelMarcey**
# Pull request motivation(s)

The URLs pointed to the live Docusaurus site for this config. Just changing it back to the demo site.

### CODE REVIEW

File: configs/docusaurus_demo.json [hunk 1]
```diff
@@ -59,4 +59,4 @@
     "415526327"
```
Comment (JoelMarcey) [inline configs/docusaurus_demo.json]: I wasn't sure if this needed to be changed -- is is docusuarus specific, opposed to docusaurus-demo?

File: configs/docusaurus_demo.json [hunk 2]
```diff
@@ -59,4 +59,4 @@
...
   ],
   "nb_hits": 1014
```
Comment (JoelMarcey) [inline configs/docusaurus_demo.json]: This too? Does this need to be changed to for docusaurus_demo?

File: configs/docusaurus_demo.json (hunk 1 above)
Comment (s-pace) [reply configs/docusaurus_demo.json]: It is docusaurus specific. You do not need to change it.

File: configs/docusaurus_demo.json (hunk 2 above)
Comment (s-pace) [reply configs/docusaurus_demo.json]: It will be changed automatically based on how many hits this PR will introduce

**Comment by s-pace** [comment]
👋 @JoelMarcey 

Thank you for the update.

I have some feedback:

- It looks like [your sitemap isn't so far updated](https://joelmarcey.github.io/docusaurus-demo/sitemap.xml)

- There is no languages for this website so I have removed it.

- Could you do a redirection from the general doc root to one of the documentation pages pls ?:
```https://joelmarcey.github.io/docusaurus-demo/docs/ -> https://joelmarcey.github.io/docusaurus-demo/docs/doc1.html``` (this stands for Docusaurus too)


I have updated it and I would be happy to receive any of your feedback regarding DocSearch.

**Comment by JoelMarcey** [comment]
Hi @s-pace -- since this is a demo, I am keeping the site pretty simple with just sample docs. I just updated the site to Docusaurus 1.2.1, and the sitemap should have been updated with it.

Do things look good from your end now?

**Comment by s-pace** [comment]
We need some small changes. I will do it today.

**Comment by s-pace** [comment]
👋 @JoelMarcey @endiliey 

In order to prevent stoping the Docusaurus search I have made a temproray index: [`docusarus_meta`](https://github.com/algolia/docsearch-configs/blob/master/configs/docusaurus_meta.json). It looks fantastic. I will let you try it out:

```html
<!-- at the end of the HEAD -->
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />

<!-- at the end of the BODY -->
<script type="text/javascript" src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
<script type="text/javascript"> docsearch({
  apiKey: 'c53337307f1787a9199348cd8263f666',
  indexName: 'docusaurus_meta',
  inputSelector: '### REPLACE ME ####',
  algoliaOptions: { 'facetFilters': ["version:$VERSION", "language:$LANGUAGE"] },
  debug: false // Set debug to true if you want to inspect the dropdown
});
</script>
```

- Replace `$VERSION` with the version you want to search on.
   The list of possible version is hardcoded in the config.
   So as of today you have: `1.0.11`, `1.0.12`, `1.0.13`, `1.0.14`, `1.0.15`, `1.1.0`, `1.1.1`, `1.1.2`, `1.1.3`, `1.1.4`, `1.1.5`, `1.2.0`, `1.2.1`, `next`

- Replace `$LANGUAGE` with the language you want to search on.
   The list of possible language is hardcoded in the config.
   So as of today you have: `en`, `es-ES`, `ro`, `tr`, `zh-CN`

  For example if you want to refine the search to the version "1.0.11" and the language "en" just specify:
```js
 'facetFilters': ["version:1.0.11", "language:en"]
```

Please note that you can define several values for every facets. It can be an array (Might be useful for `latest` for example)

Happy to give further details.

Have a nice day :)

**Comment by endiliey** [comment]
@s-pace
Is this related to the docsearch metadata we added recently ? Seems to me that now searching docs versions & language is very refined now.

Should we try this in Docusaurus @joelmarcey ?
```
'facetFilters': ["version:VERSION", "language:LANGUAGE"]
```

**Comment by s-pace** [comment]
Yes it is directly because of the meta. We use them in order to find the right context of the page. It is now a [facetFilters](https://www.algolia.com/doc/api-reference/api-parameters/facetFilters/)

**Pull request merged**